src/main.rs

```diff
@@ -34,6 +34,7 @@ fn run() -> i32 {
     opts.optopt("i", "id", "Window to capture", "ID");
     opts.optopt("g", "geometry", "Area to capture", "WxH+X+Y");
     opts.optopt("f", "format", "Output format", "png/pam");
+    opts.optflag("s", "screen", "Capture the current screen");
```

Comment:
I think something along the lines of `single-screen` or `current-screen` might be a better long name for this option. `--screen` isn't very descriptive when reading a script.